subworkflows/local/bgc.nf

```diff
@@ -131,6 +151,13 @@ workflow BGC {
         BGC_HMMER_HMMSEARCH ( ch_in_for_bgc_hmmsearch )
         ch_versions = ch_versions.mix(BGC_HMMER_HMMSEARCH.out.versions)
     }
+
+    // COMBGC
+    COMBGC ( ch_bgcresults_for_combgc )
+
+    ch_combgc_summaries = COMBGC.out.tsv
+        .collectFile(name: 'combgc_complete_summary.csv', storeDir: "${params.outdir}/reports/combgc", keepHeader:true)
```

Comment:
You are collecting `.tsv` files but emitting a `.csv` summary, is that intended?
```suggestion
        .collectFile(name: 'combgc_complete_summary.tsv', storeDir: "${params.outdir}/reports/combgc", keepHeader:true)
```